packages/tools/devtools/devtools-view/src/ThemeHelper.ts

```diff
@@ -9,6 +9,12 @@ import {
 	Theme,
 } from "@fluentui/react-components";
 
+teamsHighContrastTheme.colorSubtleBackgroundHover = "#1aebff";
```

Comment:
We'll want to get these changes reviewed from an accessibility requirements standpoint. @kashms do we have a contact for this sort of thing?

Also, how confident are we that these changes won't create color conflicts in other situations going forward?